lms/services/lti_outcomes.py

```diff
@@ -126,6 +129,7 @@ def _send_request(self, request_body):
             return self._get_body(data)
         except LTIOutcomesAPIError as err:
             err.response = response
+            log.exception(err)
```

Comment:
Does passing an `LTIOutcomesAPIError` option as the `msg` argument to `log.exception` work? The [Python docs](https://docs.python.org/3/library/logging.html#logging.Logger.exception) and [Sentry docs](https://docs.sentry.io/platforms/python/guides/logging/) to me suggests that the argument should be a string and then exception details will be captured automatically.

Reply:
Awe so if I understand that correctly, just log a message with .exception and the error will follow too.

Reply:
Yes, exactly.